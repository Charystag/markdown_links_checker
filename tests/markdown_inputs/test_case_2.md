# Test Case 2: Valid URLs with Query Parameters

This test case includes valid URLs with query parameters.

## Example 1: Links with Simple Query Strings

- [YouTube Search](https://www.youtube.com/results?search_query=openai)
- [Google Search](https://www.google.com/search?q=markdown+syntax)

## Example 2: Links with Multiple Query Parameters

- [Bing Search](https://www.bing.com/search?q=markdown&format=rss)
- [Yahoo Finance](https://finance.yahoo.com/quote/AAPL?p=AAPL&.tsrc=fin-srch)

