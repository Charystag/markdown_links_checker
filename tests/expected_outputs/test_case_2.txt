Checking file: `test_case_2.md'
[0;32mLine: `7'. url: `https://www.youtube.com/results?search_query=openai' is valid. response_code: `200'[0m
[0;32mLine: `8'. url: `https://www.google.com/search?q=markdown+syntax' is valid. response_code: `200'[0m
[0;32mLine: `12'. url: `https://www.bing.com/search?q=markdown&format=rss' is valid. response_code: `200'[0m
[0;32mLine: `13'. url: `https://finance.yahoo.com/quote/AAPL?p=AAPL&.tsrc=fin-srch' is valid. response_code: `200'[0m
[0;32mbroken links: 0/4[0m
